### ISSUE TITLE
Problem with boundary checking when passing multiple numpy arrays

### ISSUE THREAD

**Issue by tobiask-hfs** [lmrtfy/lmrtfy#20]
With the function specified here, the bounds of the first array are not checked when submitting an input file and the function runs through. If the bounds of the second or third array are violated, then the job is not executed. 

```
import numpy as np
from lmrtfy import variable, result


def add_arrays(arr_0: np.ndarray, arr_1: np.ndarray, arr_2: np.ndarray = np.array([0, 0, 0])) -> np.ndarray:
    print("Hello World!")
    return arr_0 + arr_1 + arr_2


if __name__ == "__main__":
    arr0 = variable(np.array([0, 0, 0]), name="array_0", min=0, max=10)
    arr1 = variable(np.array([0, 0, 0]), name="array_1", min=0, max=10)
    arr2 = variable(np.array([0, 0, 0]), name="array_2", min=0, max=10)

    res = add_arrays(arr0, arr1, arr2)
    res = result(res, name="add arrays")
```

The input.json file:
```
{
  "job_parameters": {
    "array_0": [
      -1, -1, -1
    ],
    "array_1": [
      1, 2, 3
    ],
    "array_2": [
      1, 2, 3
    ]
  },
  "parameter_types": {
    "array_0" : "int_array",
    "array_1" : "int_array",
    "array_2" : "int_array"
  },
  "parameter_units": {}
}
```

Job submission:
```
2022-09-07 16:06:03 [29343] INFO Start LMRTFY command line interface.
2022-09-07 16:06:03 [29343] INFO Authenticating for LMRTFY.
2022-09-07 16:06:03 [29343] INFO Validating auth token.
2022-09-07 16:06:04 [29343] INFO Auth token accepted.
2022-09-07 16:06:04 [29343] INFO Valid access token found. Login not necessary.
2022-09-07 16:06:04 [29343] INFO Submitting job with parameters: 
2022-09-07 16:06:04 [29343] WARNING {
2022-09-07 16:06:04 [29343] WARNING     "job_parameters": {
2022-09-07 16:06:04 [29343] WARNING         "array_0": [
2022-09-07 16:06:04 [29343] WARNING             -1,
2022-09-07 16:06:04 [29343] WARNING             -1,
2022-09-07 16:06:04 [29343] WARNING             -1
2022-09-07 16:06:04 [29343] WARNING         ],
2022-09-07 16:06:04 [29343] WARNING         "array_1": [
2022-09-07 16:06:04 [29343] WARNING             1,
2022-09-07 16:06:04 [29343] WARNING             2,
2022-09-07 16:06:04 [29343] WARNING             3
2022-09-07 16:06:04 [29343] WARNING         ],
2022-09-07 16:06:04 [29343] WARNING         "array_2": [
2022-09-07 16:06:04 [29343] WARNING             1,
2022-09-07 16:06:04 [29343] WARNING             2,
2022-09-07 16:06:04 [29343] WARNING             3
2022-09-07 16:06:04 [29343] WARNING         ]
2022-09-07 16:06:04 [29343] WARNING     },
2022-09-07 16:06:04 [29343] WARNING     "parameter_types": {
2022-09-07 16:06:04 [29343] WARNING         "array_0": "int_array",
2022-09-07 16:06:04 [29343] WARNING         "array_1": "int_array",
2022-09-07 16:06:04 [29343] WARNING         "array_2": "int_array"
2022-09-07 16:06:04 [29343] WARNING     },
2022-09-07 16:06:04 [29343] WARNING     "parameter_units": {}
2022-09-07 16:06:04 [29343] WARNING }
2022-09-07 16:06:04 [29343] INFO Job submission successful.
2022-09-07 16:06:04 [29343] INFO Job-id: 615f16cf-e288-4827-9c25-9809e7f8de5d

```

**Comment by orgarten** [comment]
This seems to be an issues with the profile creation, because the `min` value is not set (@cstatz). Could you please check in `~/.lmrtfy/profiles` if this profile looks like this (without the paths and meta data):

```yaml
results:
  add arrays:
    dtype: int_array
variables:
  array_0:
    dtype: int_array
    max: 10
  array_1:
    dtype: int_array
    max: 10
  array_2:
    dtype: int_array
    max: 10
```

**Comment by orgarten** [comment]
Should be fixed in PR #21 . Will be part of the next release which is likely going to happen in the next couple of days. This issue will be closed when PR #21 is merged.
